.github/workflows/commit-queue.yml

```diff
@@ -32,6 +32,7 @@ jobs:
     steps:
       - name: Get Pull Requests
         id: get_mergeable_prs
+        # TODO: the filtering below also check for the correct number of approvals
```

Comment:
I'm also piggybacking this TODO in this PR, as I imagine that we'd want the correct number of approvals to also be taken into consideration? (or I'm mistaken and it's considered already?)